frontend/src/TaskDetail.svelte

```diff
@@ -335,6 +343,10 @@ window.addEventListener('hashchange', goToSelectedLines);
             <span class="badge bg-primary" title="Automation comments" style="font-size: 60%;"
               >{comments.automated}</span>
           {/if}
+          {#if comments.summary > 0}
```

Comment:
We have a button for switching between automated/user/all comments:
<img width="30" height="45" alt="image" src="https://github.com/user-attachments/assets/9858157b-9dad-4a0f-9458-473cbafcda3c" />

It should be updated to also handle summary comments.

Although, I'm not really sure if we need to distinguish automated and summary comments.. I would treat both as "automated", to avoid introducing yet another comment category, it's already quite complicated.